Src/WitsmlExplorer.Frontend/components/ContentViews/Charts/LogsGraph.tsx

```diff
@@ -0,0 +1,307 @@
+import * as echarts from "echarts";
+import { CustomSeriesRenderItem, CustomSeriesRenderItemAPI, CustomSeriesRenderItemParams } from "echarts";
+
+import React, { useContext, useEffect, useState } from "react";
+import NavigationContext from "../../../contexts/navigationContext";
+import OperationContext from "../../../contexts/operationContext";
+import LogObject from "../../../models/logObject";
+import { calculateLogTypeId, calculateLogTypeTimeId } from "../../../models/wellbore";
+import formatDateString from "../../DateFormatter";
+import { ContentTableRow } from "../table";
+import { ReactEChartsProps, ReactLogChart } from "./ReactLogChart";
+
+export interface LogObjectRow extends ContentTableRow, LogObject {
+  logObject: LogObject;
+}
+
+export interface DataItem {
+  key: number;
+  name: string;
+  value: number[];
+  uid: string;
+  itemStyle: { normal: { color: string } };
+}
+
+interface MyCoordSys {
+  height: number;
+  type: string;
+  width: number;
+  x: number;
+  y: number;
+}
+
+interface LogItem {
+  name: string;
+  start: number;
+  end: number;
+  uid: string;
+}
+
+export const LogsGraph = (): React.ReactElement => {
+  const dimItemIndex = 0;
+  const dimStart = 1;
+  const dimEnd = 2;
+  const cartesianXBounds = [];
+  const cartesianYBounds = [];
+  const data: DataItem[] = [];
+  const categories: number[] = [];
+  const { navigationState } = useContext(NavigationContext);
+  const { selectedWellbore, selectedLogTypeGroup } = navigationState;
+
+  // dedicated colors for items containing specific names
+  const reservedColours: Map<string, string> = new Map([
+    ["#0000FF", "surface"],
+    ["#FF0000", "downhole"],
+    ["#FFFF00", "memory"]
```

Comment:
The color for "memory" makes the white log name label unreadable.

Reply:
Discussion with Asbjørn is needed.